recipe/meta.yaml

```
@@ -1,36 +1,46 @@
-{% set version = "16.2.0" %}
+{% set name = "path" %}
+{% set version = "17.1.1" %}
 
 package:
-  name: path
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/p/path/path-{{ version }}.tar.gz
-  sha256: 2de925e8d421f93bcea80d511b81accfb6a7e6b249afa4a5559557b0cf817097
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2dfcbfec8b4d960f3469c52acf133113c2a8bf12ac7b98d629fa91af87248d42
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: true  # [py<39]
 
 requirements:
   host:
     - python
     - pip
-    - wheel
-    - setuptools >=56
+    - setuptools >=77
     - setuptools_scm >=3.4.1
-    - toml
   run:
-    - python >=3.6
+    - python
 
+{% set tests_to_deselect = "" %}
+# raise NotImplementedError("Ownership not available on this platform.")
+{% set tests_to_deselect = tests_to_deselect + " --deselect=tests/test_path.py::TestOwnership::test_get_owner" %}  # [win]
+# test_no_dependencies: subprocess.CalledProcessError: Command '['$PREFIX/bin/python', '-S', '-c', 'import path']' returned non-zero exit status 1.
+{% set tests_to_deselect = tests_to_deselect + " --deselect=tests/test_path.py::test_no_dependencies" %}
 test:
+  source_files:
+    - tests
   imports:
     - path
   requires:
     - pip
+    - pytest >=6,!=8.1
+    - more-itertools
   commands:
     - pip check
+    - python -c "from importlib.metadata import version; assert(version('{{ name }}')=='{{ version }}')"
+    - pytest tests {{ tests_to_deselect }}
```

```suggestion
    - pytest -v tests {{ tests_to_deselect }}
```
Not necessary but good to have